using-mobile-extensions/adobe-target/target-event-reference.md

```diff
@@ -162,6 +163,22 @@ Here are the key-value pairs in this event:
 | `tntid` | String | Yes | The Target Identifier retrieved from Target server. |
 | `thirdpartyid` | String | Yes | Current Third Party ID that is set from the API. |
 
+### Target Preview Response
+
+This event is generated when Target Preview is started or stopped by the marketer.
+It can be used to trigger any custom workflows or API calls based on the preview flag by the following:
+
+* The original requester.
+* To any extension that might be listening for updates.
+
+#### Data payload definition
+
+Here are the key-value pairs in this event:
+
+| **Key** | **Value Type** | **Optional** | **Description** |
+| :--- | :--- | :--- | :--- |
+| `ispreviewinitiated` | Boolean | No | Value indicating whether Target preview is initiated or stopped |
```

Comment:
Please add a period at the end of the "Description" sentence.

Reply:
done